setup.py

```diff
@@ -17,13 +17,13 @@
     test_dependencies = []
 
 else:
-    dependencies = ["matplotlib>=1.2.0",
+    dependencies = ["matplotlib>=1.2.0,<1.9.0",
                     "netcdf4>=1.0",
                     "numpy",
                     "scipy>=0.15.0",
-                    "iris>=1.8.0",
-                    'psutil>=2.0.0',
-                    'six']
+                    "scitools-iris>=1.8.0",
```

Comment:
We need to use the `pypi` name for `iris`, which unfortunately is `scitools-iris`...